posts/month-in-mathlib-nov-2021.md

```diff
@@ -0,0 +1,69 @@
+---
+author: 'Mathlib community'
+category: 'month-in-mathlib'
+date: 2021-12-01 06:58:04+00:00
+description: ''
+has_math: true
+link: ''
+slug: month-in-mathlib-nov-2021
+tags: ''
+title: This month in mathlib (Nov 2021)
+type: text
+---
+
+This post summarizes some of the activity that happened in mathlib in November.
+
+## Highlighted PRs
+
+* Differential equations.
+  - [PR #9791](https://github.com/leanprover-community/mathlib/pull/9791) Prove Picard-Lindelöf/Cauchy-Lipschitz Theorem
+
+* Combinatorics.
+  - [PR #10029](https://github.com/leanprover-community/mathlib/pull/10029) Hindman's theorem on finite sums
+
+* Measure theory.
+  - [PR #9920](https://github.com/leanprover-community/mathlib/pull/9920) Define uniform pmf on an inhabited fintype
+  - [PR #10070](https://github.com/leanprover-community/mathlib/pull/10070) More on Radon-Nikodym derivatives
+  - [PR #10072](https://github.com/leanprover-community/mathlib/pull/10072) An ae measurability criterion for ennreal-valued functions
+  - [PR #10057](https://github.com/leanprover-community/mathlib/pull/10057) Define Vitali families
+  - [PR #10101](https://github.com/leanprover-community/mathlib/pull/10101) Differentiation of measures
+
+* Analysis.
+  - [PR #10145](https://github.com/leanprover-community/mathlib/pull/10145) Introduce `C*`-algebras
+  - [PR #10390](https://github.com/leanprover-community/mathlib/pull/10390) Define spectrum and prove basic prope...
+  - [PR #9840](https://github.com/leanprover-community/mathlib/pull/9840) Rayleigh quotient produces eigenvectors
```

Comment:
It's my own work, but I would like to mention the two PRs on diagonalization, part of the same project as the Rayleigh quotient PR:
```suggestion
  - [PR #9840](https://github.com/leanprover-community/mathlib/pull/9840) Rayleigh quotient produces eigenvectors
  - [PR #9995](https://github.com/leanprover-community/mathlib/pull/9995) diagonalization of self-adjoint endomorphisms (finite dimension)
  - [PR #10317](https://github.com/leanprover-community/mathlib/pull/10317) more concrete diagonalization theorem